### ISSUE TITLE
Evaluate loss function with string value

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/tensorflow-caney#8]
Evaluate loss function with string value - evaluate a function object from the yaml file for the loss
